src/settings.js

```diff
@@ -275,6 +275,12 @@ var IGNORE_CLOSURE_COMPILER_ERRORS = 0;
 // [compile+link]
 var INLINING_LIMIT = 0;
 
+// If set to 1, perform acorn pass that converts each HEAP access into a
+// function call that uses DataView to enforce LE byte order for HEAP buffer;
+// This makes generated JavaScript run on BE as well as LE machines. (If 0, only
+// LE systems are supported). Does not affect generated wasm.
+var LE_HEAP = 0;
```

Comment:
How about `BIG_ENDIAN` for the name? I don't think `LE` is clear enough, and "heap" doesn't seem necessary to add (endianness is always about memory).

Reply:
Maybe SUPPORT_BIG_ENDIAN, because if SUPPORT_BIG_ENDIAN=1 both BE and LE machines can run generated code, while if SUPPORT_BIG_ENDIAN=0 only LE are supported (same as now)

Reply:
Renamed